### DISCUSSION TITLE
Request for trigger icon

### DISCUSSION THREAD

**Discussion by jess-watson** [twilio-labs/paste#3908] [Request: Icon] [answered]
### Description

Our team is introducing the concept of triggers to journeys, which will enable customers to choose how users enter the journey. I'm using existing icons to denote each trigger type, but I'm looking to add a lightning bolt icon to represent the concept of a trigger.

Streamline has two lightning icons: [icon 1](https://www.streamlinehq.com/icons/streamline-regular/interface-essential/connect/flash) and [icon 2](https://www.streamlinehq.com/icons/streamline-regular/interface-essential/connect/flash-1)

<img width="723" alt="Screenshot 2024-05-14 at 3 28 17 PM" src="https://github.com/twilio-labs/paste/assets/92825063/3132d530-3491-4b41-802a-4b1a8aa589b1">
<img width="729" alt="Screenshot 2024-05-14 at 3 35 22 PM" src="https://github.com/twilio-labs/paste/assets/92825063/d438a726-288e-4e71-a0e0-079635ebb932">
<img width="753" alt="Screenshot 2024-05-14 at 3 36 33 PM" src="https://github.com/twilio-labs/paste/assets/92825063/132bc425-5fc0-4189-88d8-f498203b2eb4">

### Link to icon

https://www.streamlinehq.com/icons/streamline-regular/interface-essential/connect/flash

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by roxanagomez** [comment]
Hi @jess-watson , can you please [file a dependency](https://airtable.com/shrvMvL3BG6Zm0f9l) for the icon?

Also keep in mind all icon additions will now be batched together in the last sprint of each release, you can check our [release dates here](https://paste.twilio.design/roadmap) 

Let us know if there is anything else we can help with.

**Reply by roxanagomez** [reply]
Hey @jess-watson checking back on this request

**Reply by serifluous** [reply]
^ Never mind! We found the request in our Slack channel—just got lost for a bit. We'll post back here when we have updates or questions.

**Comment by serifluous** [comment, marked as answer]
Hey @jess-watson ! The icon is now available in Paste. Check it out here: https://paste.twilio.design/components/icon
